### DISCUSSION TITLE
Hello !! I'm starting to use Antmedia server and have a few questions that I hope you can answer:

### DISCUSSION THREAD

**Discussion by dukyto** [ant-media/Ant-Media-Server#5245] [FAQs]
Question 1: I rent a server with a configuration like this:

Bandwidth Plan: | 10Gbps unmetered
CPU:AMD Ryzen 9 5950X
RAM:	128GB
HDD/SSD: 2x 2TB NVME

![1](https://github.com/ant-media/Ant-Media-Server/assets/54923664/4e41076f-6208-4715-9b27-48c23aa35736)
To take full advantage of the 10Gbps network port, will the hard drive affect the live streaming process in addition to the CPU and Ram?
I plan to stream at 360 resolution and bitrate: 500kbs, will such a configuration run at full capacity of the 10Gbps network port?

Question 2: Is there any way to prevent DDOs and hide server IP? for example, if using cloudflare, will there be an increase in latency and how is the bandwidth cost calculated?

**Comment by USAMAWIZARD** [comment]
**will the hard drive affect the live streaming process in addition to the CPU and Ram?**
are you using a Hdd or nvme ssd ?
If the load is very high the hard drive may effect performance in case on HLS and DASH if you are using WebRTC it won't have any effect because WebRTC does not store file on the disk.

**Reply by klasadoroty** [reply]
Hi,
Having such a server - how many simultaneous connections will it accommodate - HLS, 1080p

**Comment by klasadoroty** [comment]
Can I get information - how many AMS can /can/ accommodate viewers?

**Reply by USAMAWIZARD** [reply]
16 core server can accommodate somewhere around  4000 HLS viewers But it also depends on the bandwidth of the server
